### DISCUSSION TITLE
Feature request: Please consider one-time donation rewards, like username fx.

### DISCUSSION THREAD

**Discussion by scooterbaga** [night/betterttv#6879] [Ideas]
Hello,
I have zero interest in a subscription service. However, I'd totally throw some money for some permanent rewards.

i.e. I would've 100% donated back in the day for a glowing username in chat. I'd like something like that, that isn't tied to a subscription.

**Comment by night** [comment]
Thank you for your feedback. I understand your concern, and I appreciate your support. Subscriptions help us maintain a steady and predictable source of income, allowing us to plan and allocate resources more effectively. As a parallel, consider sites like Wikipedia that aggressively ask for donations each year. While donations are valuable, subscriptions enable us to more easily sustain our operations.
